### PR TITLE
a11y

### PR DESCRIPTION
**Pull request by falconx**
## Description

See commit messages.

### CODE REVIEW

File: src/components/Carousel/CarouselItem.js [hunk 1]
```diff
@@ -14,6 +14,8 @@ const CarouselItem = ({ children }) => {
       {children}
       <style jsx>{`
         .carousel__item {
+          display: flex;
+          flex-direction: column-reverse;
```
Comment (Nikomus) [inline src/components/Carousel/CarouselItem.js:18]: I've always thought the name should go above the text, but neat solution!

File: src/components/CookieConsentBanner.js [hunk 2]
```diff
@@ -99,9 +108,9 @@ const CookieConsentBanner = () => {
               <Text transparent type="secondary">
                 Allowing us to use cookies does not allow us to identify you.
                 For more information please see our{` `}
-                <a href="https://www.neontribe.co.uk/privacy-policy">
+                <InternalLink to="/privacy-policy">
```
Comment (Nikomus) [inline src/components/CookieConsentBanner.js:111]: This removes the underline - a backwards step for not being differentiable by colour alone?

Comment (falconx) [reply src/components/CookieConsentBanner.js:111]: Good catch. I had assumed it was already applied by `InternalLink` or global styles to enforce consistency.

File: src/components/Carousel/CarouselItem.js (hunk 1 above)
Comment (falconx) [reply src/components/Carousel/CarouselItem.js:18]: Same. But yeah I'm a big fan of flex ordering to preserve the document hierarchy :slightly_smiling_face:

**Comment by falconx** [comment]
Closed for https://github.com/neontribe/www/pull/462

**Pull request closed without merging**